src/constants/unlimitedAnnouncement.ts

```diff
@@ -0,0 +1,9 @@
+export enum UnlimitedAnnouncementDialogSteps {
+  Announcement = 'Announcement',
+  MarketListings = 'MarketListings',
+  MegaVault = 'MegaVault',
+  AffiliatesProgram = 'Affiliates',
+  Incentives = 'Incentives',
+}
+
+export const unlimitedAnnouncementStepOrder = Object.values(UnlimitedAnnouncementDialogSteps);
```

Comment:
lol is this safe? I'm not sure we are guaranteed key ordering with normal objects.